lib/libc/picolibc/CMakeLists.txt

```diff
@@ -1,7 +1,15 @@
 # SPDX-License-Identifier: Apache-2.0
 
 zephyr_library()
-zephyr_library_sources(libc-hooks.c)
+zephyr_library_sources(
+  assert.c
```

Comment:
A nit: Before from the file name it was clear what this was. For the shake of ease of understanding of whomever wanders thru here, would it make sense to either put this in a hooks subfolder or add a comment here and/or in the files?

Reply:
Hrm. These are *all* hooks required to map between Zephyr and picolibc; that's what this directory does.